Pipfile

```diff
@@ -4,6 +4,10 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+pandas = "*"
+tqdm = "*"
+numpy = "*"
+typing-extensions = "*"
```

Comment:
Why have you added this? The [description on PyPI](https://pypi.org/project/typing-extensions/) says it's a backport of `typing`, but we're using Python 3.9.

Reply:
Hm, docker build failed without it, raising an error about `typing-extensions` missingness. I'll remove and see what happens again.